airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/TestSource.java

```diff
@@ -310,8 +310,18 @@ public void testEmptyStateIncrementalIdenticalToFullRefresh() throws Exception {
   private ConfiguredAirbyteCatalog withSourceDefinedCursors(ConfiguredAirbyteCatalog catalog) {
     ConfiguredAirbyteCatalog clone = Jsons.clone(catalog);
     for (ConfiguredAirbyteStream configuredStream : clone.getStreams()) {
-      if (configuredStream.getSyncMode() == INCREMENTAL && configuredStream.getStream().getSourceDefinedCursor()) {
-        configuredStream.setCursorField(configuredStream.getStream().getDefaultCursorField());
+      if (configuredStream.getStream().getSupportedSyncModes().contains(io.airbyte.protocol.models.SyncMode.INCREMENTAL)) {
```

Comment:
we can. would prefer to be explicit since we have two enums with the same name floating around. it's hard for me to remember which is which if one is `EnumName.BLAH` and the other is `BLAH`